app/src/test/java/com/duckduckgo/app/tabs/ui/TabSwitcherViewModelTest.kt

```diff
@@ -282,13 +292,99 @@ class TabSwitcherViewModelTest {
 
     @OptIn(ExperimentalCoroutinesApi::class)
     @Test
-    fun whenOnDraggingStartedThePixelSent() = runTest {
+    fun whenOnDraggingStartedAnnouncementDismissedAndThePixelSent() = runTest {
         testee.onTabDraggingStarted()
 
         advanceUntilIdle()
 
+        verify(mockTabRepository).setWasAnnouncementDismissed(true)
+
         val params = mapOf("userState" to NEW.name)
         verify(mockPixel).fire(AppPixelName.TAB_MANAGER_REARRANGE_TABS, params, emptyMap(), COUNT)
```

Comment:
Are these the pixels we want to check or is there a specific one for announcement dismissed we want to check as well?

Reply:
Yeah, you’re right. Previously there were no banner-specific pixels, but now there are and the drag & drop and dismissing can be tested separated. I’ve refactored this and added a few more tests.